tailwind.config.js

```diff
@@ -18,10 +27,95 @@ module.exports = {
       colors: {
         'qwestive-white': '#FAF9FA',
         'qwestive-gray': '#A3A171',
-        'qwestive-purple': '#6560fb',
+        'qwestive-purple': '#6560fb', // very near indigo 500
         'qwestive-light-blue': '#e0f2fe',
         'qwestive-purple-hover': '#4c48c2',
         'landing-page-card-gray': '#ECEFF1',
+        amber: {
```

Comment:
do we need all these colors below? Dont't see them being used

Reply:
true, it was for testing, but with jason we are making the landing page a bit different, I ll keep it until then, will remove unused colors